.changeset/old-actors-learn.md

```diff
@@ -0,0 +1,5 @@
+---
+'create-astro': minor
+---
+
+Add a question to ask the user which adapter to add. Add an `--add` flag taking a list of integration.
```

Comment:
Need to talk about TS changes as well

Reply:
I think `--add` should be split to a separate PR to avoid blocking on this review. Sounds it like we can release that independently.